packages/common-all/data/dendron-yml.validator.json

```diff
@@ -753,6 +753,9 @@
         "enableEditorDecorations": {
           "type": "boolean"
         },
+        "enableSmartRefs": {
```

Comment:
I'm not as keen on this name as I don't think it represents what the setting is doing.  I don't think one ref is 'smarter' or 'dumber' than the other, it's primarily just a different syntax.  So my preference would be like 'EnableHeaderGroupedRefSyntax' / 'EnableLegacyRefSyntax' (for the inverse setting) or something along those lines.

Reply:
i can change it but i'm also planning on removing this as an option all together (we'll remove this in two weeks). its only an option to help with migration. do you think its worth the effort to change?

Reply:
nah- that's the best case :) let's leave it then